conductor_api/src/conductor/base.rs

```diff
@@ -514,23 +540,40 @@ impl Conductor {
 
     fn spawn_interface_thread(&self, interface_config: InterfaceConfiguration) -> Sender<()> {
         let dispatcher = self.make_interface_handler(&interface_config);
-        let log_sender = self.logger.get_sender();
-        let (tx, rx) = channel();
-        thread::Builder::new()
-            .name(format!("conductor-interface: {}", interface_config.id))
-            .spawn(move || {
-                let iface = make_interface(&interface_config);
-                iface.run(dispatcher, rx).map_err(|error| {
-                    let message = format!(
-                        "err/conductor: Error running interface '{}': {}",
-                        interface_config.id, error
-                    );
-                    let _ = log_sender.send((String::from("conductor"), message));
-                    error
-                })
+        let (kill_switch_tx, kill_switch_rx) = channel();
```

Comment:
can you say something (and possibly in a code comment documented) about the kill_switch? where/when does it get called?

Reply:
Nothing new here, just gave it a more descriptive name. I'll find a place to add a comment.